examples/complicated/validators.go

```diff
@@ -170,26 +160,22 @@ func (r User) Validate() error {
 		errs.AddFieldf("Dict", "less items than 2")
 	}
 	for kDict, vDict := range r.Dict {
-		_ = kDict
-		_ = vDict
 		if utf8.RuneCountInString(string(kDict)) > 64 {
-			errs.AddFieldf(fmt.Sprintf("Dict"+".%v", kDict), "longer than 64 chars")
+			errs.AddFieldf(fmt.Sprintf("Dict"+".key[%v]", kDict), "longer than 64 chars")
 		}
 		if vDict < -35 {
-			errs.AddFieldf(fmt.Sprintf("Dict"+".%v", vDict), "less than -35")
+			errs.AddFieldf(fmt.Sprintf("Dict"+".%v", kDict), "less than -35")
```

Comment:
Are you sure it will be better to see key instead of value when validating value ? In this case seems i need to repeat test and find value manually by debugging.

Reply:
Validator shouldn't return invalid value. All other types validators just say "SomeFieldName is bad because". The same case is here.